apprenticeship-prep/underrepresented-in-tech.md

```diff
@@ -84,9 +84,9 @@ If this is your first time looking at this list of resources for underrepresente
 - [AngelHack](https://angelhack.com/)
 - [Facebook Hacker Cup](https://www.facebook.com/hackercup/)
 - [Hacktoberfest](https://hacktoberfest.digitalocean.com/)
-- [1 Million Women To Tech](https://1millionwomentotech.com/)
+- [1 Million Women To Tech](https://www.facebook.com/1millionwomentotech/)
 - [Google Kickstart](https://codingcompetitions.withgoogle.com/kickstart)
-- [Cisco Internship Program](https://www.myciip.com/)
+- [Cisco Internship Program](https://www.facebook.com/WeAreCIIP/)
```

Comment:
You found the right link, but it looks like they're cancelling this program.  Can you just remove this one?